extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelRoutesCollector.java

```diff
@@ -76,7 +76,7 @@ public XMLRoutesDefinitionLoader getXmlRoutesLoader() {
                     }
                 }
             } catch (FileNotFoundException e) {
-                LOGGER.debug("No XML routes found in {}. Skipping XML routes detection.", part);
+                LOGGER.warn("Resource {} can not be parsed. Skipping XML routes detection.", part);
```

Comment:
Isn't this still confusing? FileNotFoundException suggests that no parsing happened, because the file does not exist. So the message shout rather say that the given path does not exist.

Reply:
Yes I agree. I've overcombined it all. I'll change it to "File {} can not be found".
This will work in a case that the file doesn't exist or for example if target resource is a folder.